.gitignore

```diff
@@ -104,6 +104,7 @@ travis_wait*.log
 lightning_logs
 
 a.py
+a.md
```

Comment:
The idea is similar to `a.py`.

Reply:
What is `a.py` for?

Reply:
`a.py` is for quickly testing python code.